src/custom_nodes/image_transformation/README.md

```diff
@@ -38,4 +39,7 @@ It will compile the library inside a docker container and save the results in `l
 | target_image_color_order  | Output image color order. If specified and differs from original_image_color_order, color order conversion will be performed | `BGR` |  |
 | original_image_layout  | Input image layout. This is required to determine image shape from input shape | | &check; |
 | target_image_layout  | Output image layout. If specified and differs from original_image_layout, layout conversion will be performed | | |
+| scale  | All values will be divided by this value. When `scale_values` is specified, this value is ignored. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
+| scale_values  | Scale values to be used for the input image per channel. Input data will be divided those values. Values should be provided in the same order as output image color order. [read more](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html) | | |
```

Comment:
>Input data will be divided those values.

Should be:
Input data will be divided by those values.?

Reply:
done